### PR TITLE
Add option to restart #23

### PR DESCRIPTION
**Pull request by gwicks56**
Issue #23 
Added a reboot broadcast receiver class.
Added PERMISSION : RECEIVE_BOOT_COMPLETED
Added SharedPreference Boolean "ON" -> when app starts / restarts, this is where it will check if it was running when it was shut down. If TRUE, it will automatically move toggle "switchCore" to the ON position, and sync will continue.

### CODE REVIEW

File: app/src/main/java/com/greenaddress/abcore/MainActivity.java [hunk 1]
```diff
@@ -67,6 +66,7 @@ private void postConfigure() {
         final SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(this);
         final String useDistribution = prefs.getString("usedistribution", "core");
         mTvStatus.setText(getString(R.string.stoppedturnon, useDistribution, "knots".equals(useDistribution) ? Packages.BITCOIN_KNOTS_NDK : "liquid".equals(useDistribution) ? Packages.BITCOIN_LIQUID_NDK : Packages.BITCOIN_NDK));
+        Log.d(TAG, "postConfigure: onyl place switch is changed to on?");
```
Comment (RCasatta) [inline app/src/main/java/com/greenaddress/abcore/MainActivity.java]: typo

Comment (udiWertheimer) [inline app/src/main/java/com/greenaddress/abcore/MainActivity.java]: did you mean to remove this?

File: app/src/main/java/com/greenaddress/abcore/RebootBroadCastReceiver.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+public class RebootBroadCastReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        // TODO Auto-generated method stub
```
Comment (RCasatta) [inline app/src/main/java/com/greenaddress/abcore/RebootBroadCastReceiver.java]: remove comment

File: app/src/main/java/com/greenaddress/abcore/MainActivity.java [hunk 3]
```diff
@@ -219,6 +231,7 @@ public void onReceive(final Context context, final Intent intent) {
                     break;
                 case "localonion":
                     final String onion = intent.getStringExtra(RPCIntentService.PARAM_ONION_MSG);
+                    Log.d(TAG, "onReceive: onion: " + onion);
```
Comment (udiWertheimer) [inline app/src/main/java/com/greenaddress/abcore/MainActivity.java]: did you mean to remove this?

**Comment by RCasatta** [comment]
needs rebase,

when restarting the app is started as intended but it happened some times to have the switch ON but the switch representation OFF

Screenshot_1564057109

**Comment by gwicks56** [comment]
This got too far behind master with all the latest changes, I think I am better off cancelling it and starting a new PR.

**Pull request closed without merging**